frontend/appflowy_flutter/lib/workspace/presentation/settings/widgets/settings_user_view.dart

```diff
@@ -425,24 +420,39 @@ class IconGallery extends StatelessWidget {
   }
 }
 
+typedef SetIconFunction = void Function(String iconUrl, bool isSelected);
+
 class IconOption extends StatelessWidget {
   final FlowySvgData emoji;
   final String iconUrl;
-  final Function setIcon;
+  final SetIconFunction setIcon;
```

Comment:
nit: onSelectIcon instead of setIcon, onSelectIconCallback instead of SetIconFunction